.github/workflows/clean-envs.yml

```
@@ -24,18 +24,22 @@ jobs:
           cd .github/workflows
           npm ci
 
+      - name: Write config file
+        id: write-config-file
+        env:
+          ACCESS_TOKEN: ${{ secrets.STAGING_TOKEN }}
+        run: echo '{"token":"Token ${{ env.ACCESS_TOKEN }}","telemetry":"false","organization_slug":"saleor","organization_name":"Saleor"}' > ~/.config/saleor.json
```

Use `jq` to build JSON from arbitrary values:

```suggestion
        run: jq --null-input --arg token "Token $ACCESS_TOKEN" '{"token": $token }", "telemetry": "false", "organization_slug": "saleor", "organization_name": "Saleor"}' > ~/.config/saleor.json
```